### PR TITLE
Fixes #12355: Tags in directives are ignored in POST API

### PR DESCRIPTION
**Pull request by Haypierre**
https://www.rudder-project.org/redmine/issues/12355

### CODE REVIEW

**Comment by Haypierre** [comment]
Commit modified

File: rudder-web/src/main/scala/com/normation/rudder/web/rest/RestExtractorService.scala [hunk 1]
```diff
@@ -732,6 +728,29 @@ case class RestExtractorService (
     }
   }
 
+  // this extractTags2 is exclusively used when updating TAG in the POST APi req. We want to extract tags as a List
+  // of {key1,value1 ... keyN,valueN}
+
+ private[this] def extractTags2(value:JValue): Box[Option[Tags]] = {
```
[the remark this answers is not in the document]
Comment (Haypierre) [reply rudder-web/src/main/scala/com/normation/rudder/web/rest/RestExtractorService.scala]: Yes Master, I am doing it now!

**Comment by Haypierre** [comment]
Commit modified

**Comment by Haypierre** [comment]
OK, merging this PR

**Comment by Haypierre** [comment]
OK, merging this PR

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**